nx-X11/programs/Xserver/include/dixstruct.h

```diff
@@ -160,7 +160,7 @@ extern Bool SmartScheduleIdle;
 extern Bool SmartScheduleTimerStopped;
 extern Bool SmartScheduleStartTimer(void);
 #ifdef NXAGENT_SERVER
-extern Bool SmartScheduleStopTimer(void);
+extern void SmartScheduleStopTimer(void);
```

Comment:
This commit ("Xserver/include/dixstruct.h: Fix type mismatch for SmartScheduleStopTimer() prototype.") will get dropped with my next version of this PR branch.